python/ray/serve/common.py

```diff
@@ -31,12 +31,14 @@ def __init__(self,
                  replica_config: ReplicaConfig,
                  actor_def: Optional[ActorClass] = None,
                  version: Optional[str] = None,
-                 deployer_job_id: "Optional[ray._raylet.JobID]" = None):
+                 deployer_job_id: "Optional[ray._raylet.JobID]" = None,
+                 start_time_ms: Optional[str] = None):
```

Comment:
```suggestion
                 start_time_ms: Optional[int] = None):
```

Reply:
also does this have to be optional? does it make sense to even be optional?

Reply:
Yeah, I don't really have a good reason for making it optional other than to keep things "flexible" in case we ever have a backendinfo for which a start time doesn't make sense, or if in the future we want to initialize a BackendInfo object without a start time and add a start time later.  I don't feel strongly about it